README.md

```diff
@@ -155,64 +156,60 @@ daml script --participant-config participants.json --json-api --dar .daml/dist/d
 ```
 If you would like to boostrap the marketplace with your own data, you can either change the `doSetup` function in `daml/Setup.daml`, or create your own setup function and change the `--script-name` to `MyModule:myFunction`.
 
-### Upload and deploy the bots
-Upload the following files from `target`:
+### Add the Operator Role Contract
 
-- `da-marketplace-broker-bot-0.0.X.tar.gz`
-- `da-marketplace-custodian-bot-0.0.X.tar.gz`
-- `da-marketplace-exchange-bot-0.0.X.tar.gz`
-- `da-marketplace-issuer-bot-0.0.X.tar.gz`
-- `da-marketplace-operator-bot-0.0.X.tar.gz`
+If you have **not** bootstrapped the data, you must manually add a role contract for the `UserAdmin` party.
 
-If you are using the included matching engine instead of the [Exberry Matching Integration](#exberry-matching-engine-optional):
+On the "Live Data" tab, select the `UserAdmin` party and click "Add Contract":
+![oc01_select_add](https://user-images.githubusercontent.com/71082197/101641394-b0c3d580-39ff-11eb-94bd-6198dce25b70.png)
 
-- `da-marketplace-matching-engine-0.0.X.tar.gz`
+Then, select the `Marketplace.Operator:Operator` template, and fill in `UserAdmin` for the `Operator` field, and `Public` for the `Public` field and click "Submit":
```

Comment:
Yep, not the most ideal solution but probably the best option out of what's available!